chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/RemoveCriteriaJob.java

```diff
@@ -41,23 +36,19 @@ public RemoveCriteriaJob() throws Exception {
     public void execute(final JobExecutionContext jobContext) throws JobExecutionException {
         SpringBeanAutowiringSupport.processInjectionBasedOnCurrentContext(this);
         LOGGER.info("********* Starting the Remove Criteria job. *********");
-        if (!ff4j.check(FeatureList.EFFECTIVE_RULE_DATE)) {
-            LOGGER.info(FeatureList.EFFECTIVE_RULE_DATE + " flag is off. No criteria will be removed.");
-        } else {
-            for (String criteria : CRITERIA_TO_REMOVE) {
-                try {
-                    String[] criterionElements = criteria.split(":");
-                    CertificationCriterionDTO certDto = certCriteriaDao
-                            .getByNumberAndTitle(criterionElements[0], criterionElements[1]);
-                    certDto.setRemoved(true);
-                    certCriteriaDao.update(certDto);
-                    LOGGER.info("Updated criteria " + criteria);
-                } catch (final Exception ex) {
-                    LOGGER.error("Exception updating criteria " + criteria, ex);
-                }
+        for (String criteria : CRITERIA_TO_REMOVE) {
+            try {
+                String[] criterionElements = criteria.split(":");
+                CertificationCriterionDTO certDto = certCriteriaDao
+                        .getByNumberAndTitle(criterionElements[0], criterionElements[1]);
+                certDto.setRemoved(true);
+                certCriteriaDao.update(certDto);
+                LOGGER.info("Updated criteria " + criteria);
+            } catch (final Exception ex) {
+                LOGGER.error("Exception updating criteria " + criteria, ex);
             }
-            CacheManager.getInstance().clearAll();
         }
+        CacheManager.getInstance().clearAll();
```

Comment:
Not blocking this approval, but making note that this might be a cache that needs to be dealt with in a clustered environment. Not 100% sure.